__tests__/Auth0Client/logout.test.ts

```diff
@@ -209,7 +211,7 @@ describe('Auth0Client', () => {
       await auth0.logout({ clientId: null });
 
       expect(window.location.assign).toHaveBeenCalledWith(
-        'https://auth0_domain/v2/logout?&auth0Client=eyJuYW1lIjoiYXV0aDAtc3BhLWpzIiwidmVyc2lvbiI6IjEuMjIuMiJ9'
+        expect.stringContaining('https://auth0_domain/v2/logout')
       );
```

Comment:
Could we maybe assert that `client_id` isn't in the output here so that the assertions in this test stays the same? A second assertion like the below maybe?

```
expect(window.location.assign).toHaveBeenCalledWith(
	expect.not.stringContaining('client_id')
); 
```